server/views/pages/printReport.njk

```diff
@@ -6,31 +6,29 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/panel/macro.njk" import govukPanel %}
 
-
 {% set title = "Print a copy of this report" %}
 
 {% block pageTitle %}
     {{ title }}
 {% endblock %}
 
 {% block content %}
-    <div class="govuk-grid-row">
+    <div>
         <div class='no-print'>
-            <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-l">{{ title }}</h1>
+            <h1 class="govuk-heading-l">{{ title }}</h1>
 
-                <div class="confirmed-on-report-section">
-                    <p class="govuk-body">{{ prisonerFirstAndLastName }} must be given a copy of this report
+            <div class="confirmed-on-report-section">
+                <p class="govuk-body">{{ prisonerFirstAndLastName }} must be given a copy of this report
                         by {{ expirationTime }} on {{ expirationDay }}.</p>
-                    <div class="govuk-inset-text">You must follow local processes. You might need to give a copy of the
```

Comment:
sorry about the reformatting

Reply:
Ha - no worries - it means you look guilty instead of me when someone does git blame! Seriously though - i like the html code to be correctly formatted so I cheers for doing it.